### ISSUE TITLE
gdal 1 vs gdal 2 feedstock

### ISSUE THREAD

**Issue by ocefpaf** [conda-forge/gdal-feedstock#51]
We are using these dependencies on `gdal 1.11.4` but not in `gdal 2.1.0`.

```yaml
        - expat
        - postgresql  # [not win]
        - giflib  # [not win]
        - json-c  # [not win]
        - freexl  # [not win]
        - libspatialite  # [not win]
```

Edit:

Maybe we don't need this as we are using `openjpeg`.

```yaml
        - jasper  # [not win]
```

**Comment by gillins** [comment]
Do we need jasper if we already have openjpeg?

**Comment by ocefpaf** [comment]
> Do we need jasper if we already have openjpeg?

Probably not. I am not sure what is the preferred lib for `jpeg` (or if `gdal` is OK with both defined).

**Comment by gillins** [comment]
I don't think jasper is maintained anymore...

**Comment by ocefpaf** [comment]
> I don't think jasper is maintained anymore...

That is true. My question is if there is any reason to keep building with it. I guess not!? I will prepare a PR to tackle this issue without `jasper` and let's see how it goes.

**Comment by ocefpaf** [comment]
Closed bu #75
